File: Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, id, license, driver, typeCarAccepted, seatsMaterial) :
        super().__init__(id, license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: JS/UberVan.js
class UberVan extends Car {

    constructor(id, license, driver, typeCarAccepted, seatsMaterial){
        super(id, license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

    toString(){
        super.toString();
        console.log(this.typeCarAccepted);
        console.log(this.seatsMaterial);
    }
}